logger.py

```diff
@@ -0,0 +1,14 @@
+import os
+import logging
+
+LOG_LEVEL = logging.DEBUG if not os.getenv('NO_DEBUG_MODE') else logging.INFO
+LOG_FORMAT = '[%(asctime)s] %(filename)s %(funcName)s %(lineno)d <%(levelname)s>: %(message)s'
+logging.basicConfig(format=LOG_FORMAT)
```

Comment:
I would feel more comfortable if configuring this and setting the log levels would be done in a method call that is explicitly invoked at application startup, rather than during module loading.